CPAC/pipeline/cpac_group_runner.py

```diff
@@ -1757,6 +1759,18 @@ def run_isc_group(pipeline_dir, out_dir, working_dir, crash_dir,
         if "roi" not in levels and derivative == "roi_timeseries":
             continue
 
+        if derivative == "roi_timeseries":
+            if roi_inclusion:
+                # backwards because ROI labels embedded as substrings
+                for roi_label in roi_inclusion:
+                    if roi_label in _:
```

Comment:
what `_` is supposed to be? the mask path?

Reply:
The entire preproc strategy chunk. The ROI label is always embedded in it. Also I agree on datasink- will revisit later.